src/manifolds/SymmetricPositiveDefiniteLinearAffine.jl

```diff
@@ -225,28 +247,21 @@ function get_coordinates_orthonormal!(
     @assert size(c) == (dim,)
     @assert size(X) == (N, N)
     @assert dim == div(N * (N + 1), 2)
-    e = eigen(Symmetric(p))
-    U = e.vectors
-    S = max.(e.values, floatmin(eltype(e.values)))
-    sSqrt = Diagonal(sqrt.(S))
-    pSqrt = Symmetric(U * sSqrt * transpose(U))
+    p_sqrt = spd_sqrt(p)
     k = 1
-    V = similar(p)
+    V = similar(convert(AbstractMatrix, p))
     fill!(V, zero(eltype(V)))
-
-    F = cholesky(Symmetric(p))
-
+    F = cholesky(Symmetric(convert(AbstractMatrix, p)))
```

Comment:
You could convert `p` to `AbstractMatrix` only once and assign the result to a variable.